rpm-basic-image build with 'rt' kernel fails

**Description**

I am trying to build custom image for raspberrypi-cm3. I have setup yocto environment using worrier branch. 

I am overriding following variables in local.conf:
MACHINE ?= "raspberrypi-cm3"
RPI_USE_U_BOOT = "1"
IMAGE_INSTALL_append = " kernel-image kernel-devicetree"
IMAGE_FSTYPES_remove += " rpi-sdimg"
IMAGE_FSTYPES = "ext4"
KERNEL_BOOTCMD = "bootz"
KERNEL_IMGTYPE = "zImage"
PREFERRED_PROVIDER_virtual/kernel ?= "linux-raspberrypi-rt"

Bitbake rpi-basic-image fails with following error:

NOTE: make -j 2 HOSTCC=gcc  -isystem/home/ubuntu/yocto/cros-build/tmp/work/raspberrypi_cm3-poky-linux-gnueabi/linux-raspberrypi-rt/1_4.14.81+gitAUTOINC+acf578d07d-r0/recipe-sysroot-native/usr/include -O2 -pipe -L/home/ubuntu/yocto/cros-build/tmp/work/raspberrypi_cm3-poky-linux-gnueabi/linux-raspberrypi-rt/1_4.14.81+gitAUTOINC+acf578d07d-r0/recipe-sysroot-native/usr/lib                         -L/home/ubuntu/yocto/cros-build/tmp/work/raspberrypi_cm3-poky-linux-gnueabi/linux-raspberrypi-rt/1_4.14.81+gitAUTOINC+acf578d07d-r0/recipe-sysroot-native/lib                         -Wl,--enable-new-dtags                         -Wl,-rpath-link,/home/ubuntu/yocto/cros-build/tmp/work/raspberrypi_cm3-poky-linux-gnueabi/linux-raspberrypi-rt/1_4.14.81+gitAUTOINC+acf578d07d-r0/recipe-sysroot-native/usr/lib                         -Wl,-rpath-link,/home/ubuntu/yocto/cros-build/tmp/work/raspberrypi_cm3-poky-linux-gnueabi/linux-raspberrypi-rt/1_4.14.81+gitAUTOINC+acf578d07d-r0/recipe-sysroot-native/lib                         -Wl,-rpath,/home/ubuntu/yocto/cros-build/tmp/work/raspberrypi_cm3-poky-linux-gnueabi/linux-raspberrypi-rt/1_4.14.81+gitAUTOINC+acf578d07d-r0/recipe-sysroot-native/usr/lib                         -Wl,-rpath,/home/ubuntu/yocto/cros-build/tmp/work/raspberrypi_cm3-poky-linux-gnueabi/linux-raspberrypi-rt/1_4.14.81+gitAUTOINC+acf578d07d-r0/recipe-sysroot-native/lib                         -Wl,-O1 -Wl,--allow-shlib-undefined -Wl,--dynamic-linker=/home/ubuntu/yocto/cros-build/tmp/sysroots-uninative/x86_64-linux/lib/ld-linux-x86-64.so.2 HOSTCPP=gcc  -E bcm2708-rpi-zero-w.dtb
|   CC      scripts/mod/empty.o
|   CC      scripts/mod/devicetable-offsets.s
|   MKELF   scripts/mod/elfconfig.h
|   HOSTCC  scripts/mod/modpost.o
|   CHK     scripts/mod/devicetable-offsets.h
|   HOSTCC  scripts/mod/sumversion.o
|   HOSTCC  scripts/mod/file2alias.o
|   HOSTLD  scripts/mod/modpost
| make[3]: *** No rule to make target 'arch/arm/boot/dts/bcm2708-rpi-zero-w.dtb'.  Stop.
| arch/arm/Makefile:343: recipe for target 'bcm2708-rpi-zero-w.dtb' failed

**Additional information you deem important (e.g. issue happens only occasionally):**
None
**Additional details (revisions used, host distro, etc.):**
poky - warrior branch

This is the first time I am using yocto build system and haven't yet understood all the steps. Would really appreciate, if someone could point out if there is anything wrong with this setup. 

One more thing I found is: In $Build/tmp/work/linux-raspberrypi-rt/..../arch/arm/boot/dts folder is empty, where as same exists for normal kernel build.

--- a/recipes-kernel/linux/linux-raspberrypi-rt_4.14.bb
+++ b/recipes-kernel/linux/linux-raspberrypi-rt_4.14.bb
@@ -1,6 +1,6 @@
-LINUX_VERSION ?= "4.14.81"
+LINUX_VERSION ?= "4.14.91"
 
-SRCREV = "acf578d07d57480674d5361df9171fe9528765cb"
+SRCREV = "0b520d5f1f580d36a742a9457a5673fa1578fff3"
 SRC_URI = " \
     git://github.com/raspberrypi/linux.git;branch=rpi-4.14.y-rt \
     file://0001-menuconfig-check-lxdiaglog.sh-Allow-specification-of.patch \

